connection.go

```diff
@@ -304,3 +311,33 @@ func (sc *snowflakeConn) populateSessionParameters(parameters []nameValueParamet
 		sc.cfg.Params[strings.ToLower(param.Name)] = &v
 	}
 }
+
+func (sc *snowflakeConn) isClientSessionKeepAliveEnabled() bool {
+	ret := false
+	for k, v := range sc.cfg.Params {
+		if k == sessionClientSessionKeepAlive {
+			if strings.Compare(*v, "true") == 0 {
```

Comment:
Will this always be lowercased? Also, instead of iterating through the entire map, I think you can do something like:

```
_ ok := sc.cfg.Params["sessionClientSessionKeepAlive"]
if !ok {
    return false
}
return strings.Compare(sc.cfg.Params["sessionClientSessionKeepAlive"], "true") == 0
```

Reply:
ah, ok. Yes, the value should be lower cases.

Reply:
```
	v, ok := sc.cfg.Params[sessionClientSessionKeepAlive]
	if !ok {
		return false
	}
	return strings.Compare(*v, "true") == 0
```